src/tcgwars/logic/impl/gen3/DragonFrontiers.groovy

```diff
@@ -1903,10 +1903,12 @@ public enum DragonFrontiers implements LogicCardInfo {
           text "As long as Flygon ex is your Active Pokémon, put 1 damage counter on each of your opponent's Benched Basic Pokémon between turns. You can't use more than 1 Sand Damage Poké-Body between turns."
           delayedA {
             before BEGIN_TURN, {
+              boolean flag = 1
```

Comment:
Defines boolean specifically, but then proceeds to use an integer value instead of a boolean value to assign to it :laughing:

Reply:
I blame whoever did weezing, and my brain post handling a father's day lunch videoconference video 😛